packages/parts/src/routes/2-settings/gloss-languages.svx

```diff
@@ -6,23 +6,38 @@
 </script>
 
 # Edit Gloss Languages
-<Story name="Allow removing">
+<!-- <Story name="Editable">
   <EditableGlossesField 
     admin 
     {selectedLanguages} 
     availableLanguages={glossingLanguages} 
     on:add={(e) => selectedLanguages = [...selectedLanguages, e.detail.languageId]} 
     on:remove={e => {
       selectedLanguages.splice(selectedLanguages.indexOf(e.detail.languageId), 1);
-      selectedLanguages = selectedLanguages
+      selectedLanguages = selectedLanguages;
     }} />
-</Story>
+</Story> -->
 
-<Story name="Removing not possible">
+<Story name="Removing not possible when there's only one language">
   <EditableGlossesField 
-    selectedLanguages={['es', 'en']} 
+    admin
+    {selectedLanguages} 
     availableLanguages={glossingLanguages}
-     on:add={(e) => alert(`Add a new Id: ${e.detail.languageId}`)} />
+    on:add={(e) => selectedLanguages = [...selectedLanguages, e.detail.languageId]}
```

Comment:
@jacob-8 please review this and tell me if it's fine for you. I would like to use arrays in the story input functionality you created, but I think you are trying to handle that, right?